style.css

```diff
@@ -569,3 +623,499 @@ input[type="submit"]:active {
 input[type="submit"]:disabled {
   background-color: red;
 }
+
+.grid-container5 {
+  display: none;
+}
+
+.grid-container6 {
+  display: none;
+}
+
+.grid-container7 {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  header {
+    width: 1440px;
+    display: flex;
+    justify-content: end;
+  }
+
+  header nav {
+    width: 1440px;
+    display: flex;
+    text-align: center;
+    vertical-align: center;
+    flex-direction: row;
+    justify-content: space-between;
+    color: #6070ff;
+    font-style: bold;
+  }
+
+  header .fa-solid {
+    display: none;
+  }
+
+  header h2 {
+    width: 76px;
+    height: 20px;
+    font-weight: 700;
+    font-size: 18px;
+    margin-left: 120px;
+  }
+
+  header ul {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    column-gap: 25px;
+    list-style: none;
+    font-size: 15px;
+    font-family: 500;
+    margin-right: 200px;
+  }
+
+  nav h2 {
+    width: 100px;
+    height: 100px;
+    justify-content: flex-start;
+  }
+
+  .section-1 {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    background-image: url("Header bg.png");
+    width: 1440px;
+    height: 816px;
+    margin-top: 8px;
+  }
+
+  .section-1 h1 {
+    width: 920px;
+    height: 122px;
+    font-size: 48px;
+    color: #091e42;
+    margin-top: 1px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+  }
+
+  p {
+    line-height: 24px;
+  }
+
+  form header p {
+    width: 569px;
+    height: 102px;
+    font-size: 20px;
+  }
+
+  .section-1 div {
+    width: 920px;
+    padding-left: 260px;
+    padding-right: 260px;
+    padding-top: 359px;
+    padding-bottom: 435px;
+  }
```

Comment:
- [ ] Kindly check this section of your project. There's a horizontal scroll bar at the bottom of your page. Take a look at the screenshot and try to fix it.

![footer-scrollbar](https://user-images.githubusercontent.com/26597432/194412622-bfa4befa-adab-42ad-8f45-5e469e1cd267.png)


- [ ] This also affects your mobile version. Kindly take a look and fix it.

![headline-section](https://user-images.githubusercontent.com/26597432/194412907-3eee08b3-d63a-45e8-a472-8aef8eb53783.png)

Reply:
Kindly, following the theme of the previous reviewer, your page in mobile version presents a horizontal scroll that does not look very good, I suggest you find the element that is causing this and adjust it, this will make your site look more professional.

![image](https://user-images.githubusercontent.com/41912912/194473712-3815e59f-1b4f-4acd-b667-f9ea8015f77c.png)

Reply:
@mgmediaweb I don't understand what you commented. What type of horizontal scroll?